test/all-options.test.js

```diff
@@ -33,6 +27,14 @@ describe('when applied with all options', () => {
           return `License information can be found in ${licenseFile}`;
         },
       },
+      uglifyOptions: {
```

Comment:
I felt like they were too many options to keep them at top level. (https://github.com/mishoo/UglifyJS2/tree/harmony#minify-options-structure)
That way, I thought it was easier to understand what's coming from the plugin, or what's coming from UglifyJs, and so easier to read the docs according to the need.